docs/User/Swarm_Learning_Command_Interface.md

```diff
@@ -65,19 +66,22 @@ A SWCI tool can control operations on multiple Swarm Learning framework instance
-|<code> GET TASKRUNNER PEER STATUS <taskRunnerName : string> <swopIndex&vert;swopUid></code>:|Displays current status for the specified PEER \(SWOP node\) in the Taskrunner’s context.<br>-   `taskRunnerName`: Specifies the unique name of Taskrunner.<br>-   `swopIndex`: The index number of SWOP Node, starts with 0 up to `ENROLLEDSWOP`.<br>-   `swopUid`: The UID String of the SWOP Node. You can specify either `swopIndex` or `swopUid`, not both.<br>|
+|`ASSIGN TASK <taskName : string> TO <trName : string> WITH <peersNeeded> PEERS`|Assigns a task to a taskrunner instance and specify the minimum peer count for declaring the result. This triggers SWOPs to execute this task<br>-   `taskName`: Specifies the unique name of task.<br>-   `trName`: Specifies the unique name of Taskrunner. SWOP’s listening on this Taskrunner participates and executes this task.<br>-   `WITH <peersNeeded> PEERS`:<br>`peersNeeded` is a nonzero positive integer specifying the minimum number of peers required to complete this task.<br>Meaningful value of minimum peers is dependent on the task type. For RUN\_SWARM task, it is the number of SL and ML node pairs. For all other TASK types, it is the number of SWOP nodes.<br>-  For RUN_SWARM task type, the actual numbers of SL/ML peers started could be equal or greater than `peersNeeded`, depending on the number of the SL nodes defined in the SWOP profiles.<br>- For other TASK types, the number of SWOPs participating would be equal to `peersNeeded`.|
+|`GET TASKRUNNER PEER STATUS <taskRunnerName:string><swopIndex\swopUid>:`|Displays current status for the specified PEER in the Taskrunner’s context.<br>-   `taskRunnerName`: Specifies the unique name of Taskrunner.<br>-   `swopIndex`: The index number of SWOP Node, starts with 0 up to ENROLLEDSWOP.<br>-   `swopUid:`: The UID String of the SWOP Node. You can specify either `swopIndex` or `swopUid`, not both.<br>The status of the PEER differs based on the type of the current TASK that has been assigned to TASKRUNNER.<br>  For RUN_SWARM task, the status summary reports SWOP node UID, Number of SL PEERs this SWOP has spawned, and list of all SL node information (UID, Status, Description).<br> For all other types of tasks, the status summary reports SWOP node status (UID, Status, Description).<br> **Note**: Node UID can be used to identify the container name/id from ‘LIST NODES’ command. With container name/id, user can debug the error with docker logs.|
```

Comment:
When looked into the preview mode(Display the rich diff), these 2 commands seems to have merged in single row. Add <br> at appropriate place.